projects/aca-content/assets/i18n/en.json

```diff
@@ -218,7 +218,29 @@
                 "SAVE_SUCCESS": "Search Saved",
                 "SAVE_ERROR": "Error occured. Search could not be saved.",
                 "NAVBAR": {
-                    "TITLE": "Saved Searches ({{ number }})"
+                  "TITLE": "Saved Searches ({{ number }})",
+                  "MANAGE_BUTTON": "Manage searches"
+                },
+                "EDIT_DIALOG": {
+                  "CONTEXT_OPTION": "Edit Search",
+                  "TITLE": "Edit Saved Search",
+                  "SUCCESS_MESSAGE": "Saved Search edited successfully",
+                  "ERROR_MESSAGE": "Error occured. Saved Search could not be edited."
+                },
+                "DELETE_DIALOG": {
+                  "CONTEXT_OPTION": "Delete Search",
+                  "TITLE": "Delete Saved Search",
+                  "CONTENT": "Are you sure you want to delete this search",
+                  "SUCCESS_MESSAGE": "Saved Search deleted successfully",
+                  "ERROR_MESSAGE": "Error occured. Saved Search could not be deleted."
```

Comment:
unused translation

Reply:
There was a mistake in one error handling case, it is used now